shared/src/main/scala/io/github/hamsters/FutureOps.scala

```diff
@@ -23,6 +23,22 @@ object FutureOps {
     def squash(implicit ec: ExecutionContext): Future[A] = futureEitherStack.flatMap(fromEither)
   }
 
+  /**
+    * Converts a Try[A] to a Future[A] that may raise a Throwable
+    *
+    * @param tr
+    * @tparam A
+    * @return a Future[A]
+    */
+  def fromTry[A](tr: Try[A]): Future[A] = tr match {
+    case Success(a) => Future.successful(a)
+    case Failure(th) => Future.failed(th)
+  }
+
+  implicit class FutureTry[A](futureTryStack: Future[Try[A]]) {
+    def squash(implicit ec: ExecutionContext): Future[A] = futureTryStack.flatMap(fromTry)

```

Comment:
Future.fromTry (from std lib) you be used here

Reply:
Sure, completely forgot about it)